javascript-practice/src/js/main.js

```diff
@@ -0,0 +1,11 @@
+// import UserView from "./views/userView";
```

Comment:
Remove code if you don't use it. Btw, should have only 1 empty line at the end of file